### ISSUE TITLE
Is there a config like webpack publicPath

### ISSUE THREAD

**Issue by jingegebuguai** [rollup/rollup#4632]
like vite base、webpack publicPath

**Comment by chinanderm** [comment]
I would like this as well. It is the one thing that is keeping me on webpack.

**Comment by lemanschik** [comment]
can you explain that a bit more this is not a action able itm ? i need code samples and what you expect from that

**Comment by chinanderm** [comment]
> can you explain that a bit more this is not a action able itm ? i need code samples and what you expect from that

Sure thing! We are referring to the webpack config option [Public Path](https://webpack.js.org/guides/public-path/).

My specific use case is bundling a JS "app" with code splitting for inclusion and usage on website whose URL is not the same as my JS origin. With code splitting, it needs to load the chunks from the JS origin/script source, not the origin in which the JS is being run on.

This scenario is what [Automatic publicPath](https://webpack.js.org/guides/public-path/#automatic-publicpath) webpack config solves.

To recap:
- Bundled and code-split code is hosted at `domain-a.com`
- `<script src="https://domain-a.com/script.js" />` is used at `domain-b.com`
- The chunks referenced in `domain-a.com/script.js` should load chunks from `domain-a.com/:chunk.js`
- Rollup currently tries to load them from `domain-b.com`

**Comment by lemanschik** [comment]
@chinanderm there are many ways that come in my mind to solve that rollup is using relativ pathes right=?

try add a base tag 
````
<base url=
```

this way you can configure a base url for the resolution of all relativ urls you can even place more then one baseTag

also you can use service workers to archive that and let them cache the requests. i understand your problem in general

you can also solve that via references there is a example in the large single page doc of rollup simply search for references

i guess the base url tag is a interristing thing to try it works in a lot of my cases to simply lock the domain for a script part and its resolve.
